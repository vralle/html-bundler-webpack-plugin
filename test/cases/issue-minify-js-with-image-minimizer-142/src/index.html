<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
    <link href="./style.css" rel="stylesheet">
    <script src="./main.js"></script>
  </head>
  <body>
    <h1>Hello World!</h1>
    <img src="@images/apple.png?as=webp" alt="" />
  </body>
</html>