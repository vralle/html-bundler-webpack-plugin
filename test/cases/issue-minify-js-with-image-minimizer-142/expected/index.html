<!doctype html><html><head><title>Test</title><link href="css/style.7947baa4.css" rel="stylesheet"><script src="js/main.09a3ac0e.js"></script></head><body><h1>Hello World!</h1><img src="img/apple.webp" alt=""/></body></html>